### ISSUE TITLE
tests are not working with truffle 3.4.6+

### ISSUE THREAD

**Issue by troggy** [acebusters/acebusters-economy#4]
Call to `sendTransaction` fails with a stacktrace like:

```
Error: Invalid number of arguments to Solidity function
      at Object.InvalidNumberOfSolidityArgs (/home/troggy/.nvm/versions/node/v6.10.3/lib/node_modules/truffle/build/cli.bundled.js:37967:16)
      at SolidityFunction.validateArgs (/home/troggy/.nvm/versions/node/v6.10.3/lib/node_modules/truffle/build/cli.bundled.js:231124:22)
      at SolidityFunction.toPayload (/home/troggy/.nvm/versions/node/v6.10.3/lib/node_modules/truffle/build/cli.bundled.js:231140:10)
      at SolidityFunction.sendTransaction (/home/troggy/.nvm/versions/node/v6.10.3/lib/node_modules/truffle/build/cli.bundled.js:231213:24)
      at SolidityFunction.execute (/home/troggy/.nvm/versions/node/v6.10.3/lib/node_modules/truffle/build/cli.bundled.js:231306:37)
      at /home/troggy/.nvm/versions/node/v6.10.3/lib/node_modules/truffle/build/cli.bundled.js:220999:16
      at /home/troggy/.nvm/versions/node/v6.10.3/lib/node_modules/truffle/build/cli.bundled.js:220965:18
      at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Workaround**
Works with truffle@3.4.5

**Comment by johannbarbie** [comment]
fixed in #7, as i understand

**Comment by troggy** [comment]
nope, wasn't fixed
